dashboard/app/controllers/levels_controller.rb

```diff
@@ -286,11 +286,29 @@ def level_params
       {level_concept_difficulty_attributes: [:id] + LevelConceptDifficulty::CONCEPTS},
       {soft_buttons: []},
       {contained_level_names: []},
-      {examples: []}
+      {examples: []},
+
+      # Minecraft-specific
+      {available_blocks: []},
+      {drop_dropdown_options: []},
+      {if_block_options: []},
+      {place_block_options: []},
+      {play_sound_options: []},
```

Comment:
@joshlory I'm really not sure how I feel about adding type-specific fields to this general controller, but the alternative would involve having it call a type-specific version of the `permitted_params` method which we have been specifically avoiding so far. What're your thoughts on this addition?